### ISSUE TITLE
Option to have badge number equal number of messages directed at you (incl. @channel or @here)

### ISSUE THREAD

**Issue by Vages** [ZeeZide/Shrugs#21]
Currently, a red bagde is shown in the dock if any of you channels have unread messages. The number on the badge is equal to the number of unread messages. This number quickly climbs to two or three digits for things that don't require any immediate action.

To improve the signal-to-noise ratio, users should have an option to only show important mentions that are directed at you (either through an @yourname, @here, @channel or a traditional DM). It should mimic this setting in Slack: 
<img width="808" alt="image" src="https://user-images.githubusercontent.com/3520744/95862860-03ac4400-0d53-11eb-8968-33fb87a13c37.png">

It would also be great if I could choose the same setting for the sidebar numbers (i.e. the number in the sidebar badges would be calculated using the same formula). I wouldn't mind it if these two behaviours were controlled by the same knob in the application settings. 

If I'm not mistaken, what I'm requesting mimics the default Slack behaviour.

**Comment by helje5** [comment]
Yes, Slack focuses on reporting mentions, not unreads. Which is really nice, I should have that as well.
